app/src/main/java/fr/free/nrw/commons/media/MediaDetailPagerFragment.java

```diff
@@ -134,7 +134,7 @@ public View onCreateView(LayoutInflater inflater,
 
         // If fragment is associated with ProfileActivity, then hide the tabLayout
         if (getActivity() instanceof ProfileActivity) {
-            ((ProfileActivity)getActivity()).tabLayout.setVisibility(View.GONE);
+            ((ProfileActivity)getActivity()).binding.tabLayout.setVisibility(View.GONE);
```

Comment:
I know you didnt introduce this, but the code here breaks the [Law of Demeter](https://www.baeldung.com/java-demeter-law) by accessing internal state of another class, made worse by needing to do the `instanceof` and cast.  Is there anything you could do to improve things?

Reply:
sure I'll take look at it

Reply:
@psh Just pushed a fix, please take a look at it